ext/opcache/tests/gh8846.phpt

```diff
@@ -0,0 +1,59 @@
+--TEST--
+Bug GH-8846: Delayed early binding can be used for classes without parents
+--EXTENSIONS--
+opcache
+--CONFLICTS--
+server
+--INI--
+opcache.validate_timestamps=1
+opcache.revalidate_freq=0
+--FILE--
+<?php
+
+file_put_contents(__DIR__ . '/gh8846-index.php', <<<'PHP'
+<?php
+if (!@$_GET['skip']) {
+    include __DIR__ . '/gh8846-1.php';
+}
+include __DIR__ . '/gh8846-2.php';
+echo "Ok\n";
+PHP);
+
+file_put_contents(__DIR__ . '/gh8846-1.php', <<<'PHP'
+<?php
+class Foo {
+	const BAR = true;
+}
+PHP);
+
+file_put_contents(__DIR__ . '/gh8846-2.php', <<<'PHP'
+<?php
+var_dump(Foo::BAR);
+class Foo {
+	const BAR = true;
+}
+PHP);
+
+// FIXME: Avoid this sleep
+sleep(2);
+
+include 'php_cli_server.inc';
+php_cli_server_start('-d opcache.enable=1 -d opcache.enable_cli=1');
```

Comment:
I had to load opcache explicitly here to make the test fail: 

```
php_cli_server_start('-d opcache.enable=1 -d opcache.enable_cli=1 -d zend_extension='.ini_get('extension_dir').'/opcache.so');
```

Reply:
Same here. I have `extension_dir` configured for my php dev setup, but ext/opcache/tests/php_cli_server.inc adds the `-n` flag when starting the server. It might make sense to remove that (we don't do `-n` for other tests either) but I didn't want to change this in an unrelated PR.

Reply:
I'll create a separate PR for this.